r2/service.go

```diff
@@ -536,11 +611,13 @@ func (m mappingRuleJSON) mappingRuleView() *rules.MappingRuleView {
 
 func newMappingRuleJSON(mrv *rules.MappingRuleView) mappingRuleJSON {
 	return mappingRuleJSON{
-		ID:            mrv.ID,
-		Name:          mrv.Name,
-		Filters:       mrv.Filters,
-		Policies:      mrv.Policies,
-		CutoverMillis: xtime.ToUnixMillis(xtime.FromNanoseconds(mrv.CutoverNanos)),
+		ID:                  mrv.ID,
+		Name:                mrv.Name,
+		Filters:             mrv.Filters,
+		Policies:            mrv.Policies,
+		CutoverMillis:       mrv.CutoverNanos / int64(time.Millisecond),
```

Comment:
nit: perhaps slightly more readable to do
```
nanosPerMilli = int64(time.Millisecond / time.Nanosecond)
```
and then here do
`CutoverMillis: mrs.CutoverNanos / nanosPerMilli,`